integration-tests/integration/reportPages/check-your-answers.cy.js

```diff
@@ -30,6 +30,12 @@ context('Check your answers page', () => {
     cy.login()
   })
 
```

Comment:
this verifies that in the 'straight-forward' case of a new report being created and the locationsInsidePrisonApi's location id is persisted, the correct location description is displayed in /check-your-answers